etc/systemd/system/tiny-dfr.service

```diff
@@ -7,3 +7,6 @@ StartLimitBurst=2
 [Service]
 ExecStart=/usr/bin/tiny-dfr
 Restart=always
+
+[Install]
```

Comment:
this was copied from ubuntu package.
Shall I remove it?

Reply:
Yes